internal/server/v2/facet/golden/contained_in_facet/CA_County_2015.json

```diff
@@ -1,89 +0,0 @@
-{
```

Comment:
why is this file deleted?

Reply:
it's not used in any of the tests, I think it's outdated

Reply:
I see, it was removed in https://github.com/datacommonsorg/mixer/pull/1295/files#diff-f92f6edcac465aee9633f109b36078f74067ded3b7f3fe7210be800fc60727a5. Can you add it back here?

Reply:
oh, did you remove it for a reason? or by accident?

Reply:
I can not remember, likely by accident.